src/lib/molecules/Copy/Copy.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useRef } from 'react';
+import { Meta, StoryObj } from '@storybook/react';
+import Copy, { ICopyProps } from './index';
+import { args, category } from '../../../../stories/assets/storybook.globals';
+import { Textarea } from '../../../index';
+
+const meta: React.FC<ICopyProps> = {
```

Comment:
I think it would be better 

meta։Meta<typeof Copy>
link : https://storybook.js.org/docs/configure/typescript